### PR TITLE
Make clear that `fullNameChanged` fires if `fullName` is consumed first

### PR DESCRIPTION
**Pull request by ryrych**
Fixes https://github.com/emberjs/guides/issues/893#issuecomment-148925813

### CODE REVIEW

**Comment by locks** [comment]
Hm, I actually don't like how the original example complicates the explanation.
Thoughts @michaelrkn?

File: source/object-model/observers.md [hunk 1]
```diff
@@ -30,7 +31,9 @@ var person = Person.create({
   lastName: 'Katz'
 });
 
-person.set('firstName', 'Brohuda'); // observer will fire
+// observer won't fire until `fullName` is consumed first
```
Comment (locks) [inline source/object-model/observers.md]: what do you think of switching it around to `// consume "fullName" so the observer is triggered`?

**Comment by locks** [comment]
p.s. sorry for slightly editing your post, but I think this way the issue will get closed when this gets merged. And thanks for your contribution!

**Comment by ryrych** [comment]
@locks no problem, you can switch the order :+1:

**Comment by michaelrkn** [comment]
I think it's pretty clear how it is. @locks why don't you open another PR if you think there's a better way to explain it.

**Pull request merged**